src/Pagination/components/PageNumberButton.ts

```diff
@@ -1,22 +1,52 @@
-import { SFC, createElement } from "react";
+import { ReactElement, createElement } from "react";
 import * as classNames from "classnames";
 
 interface PageNumberButtonProps {
     pageNumber: number;
     selectedPageNumber: number;
-    onClickAction: (pageNumber: number) => void;
+    onClickAction: () => void;
     key?: string | number;
 }
 
-export const PageNumberButton: SFC<PageNumberButtonProps> = (props: PageNumberButtonProps) => createElement("li", {
-        className: classNames(
-            props.selectedPageNumber === props.pageNumber ? "active" : "",
-            props.pageNumber < 10 ? "single-digit" : ""
-        ),
-        onClick: () => props.onClickAction(props.pageNumber),
-        key: props.key
-    },
-    props.pageNumber
-);
+export const PageNumberButton = (
+    props: PageNumberButtonProps
+): ReactElement => {
+    const { onClickAction, pageNumber, selectedPageNumber } = props;
+
+    return createElement(
+        "li",
+        {
+            className: classNames(
+                selectedPageNumber === pageNumber ? "active" : ""
+            ),
+            role: "button",
+            onClick: onClickAction,
+            onKeyDown: onKeyDown.bind(null, onClickAction),
+            onKeyUp: onKeyUp.bind(null, onClickAction),
+            key: props.key,
+            tabindex: 0,
+            "aria-label":
+                selectedPageNumber === pageNumber
+                    ? `Current page, page ${pageNumber}`
```

Comment:
Here you can use only this translatable key: `mxui.widget.Grid.a11y.page_status` and just say: `Currently showing x of z`

Reply:
I changed it, so that it uses translatable. I didn't add the `x of z` part, because `of` is language specific. We can't use placeholders, because this isn't supported for `mxui.widget.Grid.a11y.page_status` unfortunately.

Reply:
Removed system text page_status, because the text doesn't fit the button.